source/query-with-natural-language/ai-and-data-usage-information.txt

```diff
@@ -0,0 +1,44 @@
+.. _compass-ai-data-usage:
+
+=============================
+AI and Data Usage Information
+=============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+Querying with natural language in Compass is powered by Generative AI
+(Gen AI), and may give inaccurate responses. Please see our `Generative AI FAQ 
```

Comment:
(nit -- wording)
Stylistically, we generally omit things like please and thank you. :) 

```suggestion
(Gen AI), and may give inaccurate responses. See our `Generative AI FAQ 
```